src/researches/findKeywordInFirstParagraph.js

```diff
@@ -1,23 +1,83 @@
 /** @module analyses/findKeywordInFirstParagraph */
 
 const matchParagraphs = require( "../stringProcessing/matchParagraphs.js" );
-const wordMatch = require( "../stringProcessing/matchTextWithWord.js" );
+const getSentences = require( "../stringProcessing/getSentences.js" );
+const findTopicFormsInString = require( "./findKeywordFormsInString.js" ).findTopicFormsInString;
+const imageInText = require( "../stringProcessing/imageInText" );
 
-const escapeRegExp = require( "lodash/escapeRegExp" );
 const reject = require( "lodash/reject" );
 const isEmpty = require( "lodash/isEmpty" );
 
 /**
- * Counts the occurrences of the keyword in the first paragraph, returns 0 if it is not found,
- * if there is no paragraph tag or 0 hits, it checks for 2 newlines, otherwise returns the keyword
- * count of the complete text.
+ * Checks if the paragraph consists only of images.
+ *
+ * @param {string} text The text string to analyze.
+ *
+ * @returns {boolean} True if the text consists only of images, false otherwise.
+ */
+function paragraphConsistsOfImagesOnly( text ) {
+	const images = imageInText( text );
+	if ( images.length < 1 ) {
+		return false;
+	}
+
+	images.forEach( function( image ) {
+		text = text.replace( image, "" );
+	} );
+
+	return text === "";
+}
+
+/**
+ * First splits the first paragraph by sentences. Finds the first paragraph which contains sentences e.g., not an image).
+ * Tries to find all (content) words from the keyphrase or a synonym phrase within one sentence.
+ *    If found all words within one sentence, returns an object with foundInOneSentence = true and
```

Comment:
This sentence does not read well, particularly: "If found all words within one sentence".
Maybe "When all words are in one sentance".